1-foundations/2-control-flows-and-functions/foundations-s2.ipynb

```diff
@@ -1,5 +1,13 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d8b75c26-1e53-48bd-86d2-5812947a83fc",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb)"
```

Comment:
```suggestion
    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/worldbank/dec-python-course/blob/main/1-foundations/2-control-flows-and-functions/foundations-s2.ipynb)"
```

This button will now point to this notebook in the `main` branch. Which I think it should, but then it becomes important that we remember to merge this PR before your session